### PR TITLE
IP LogicProfile Info

### PR DESCRIPTION
**Pull request by skorpiom**
Hey,
I've written function that takes user object and if there are all demanded fields returns filled up component, otherwise returns nothing. 

Do you have any suggestions? ;)

### CODE REVIEW

File: Src/Logic/SetProfileInfo/SetProfileInfo.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { profileInfo } from "../../UIComponents/ProfileInfo/ProfileInfo";
+import {readFromLocalStorage} from '../../../Src/Logic/LocalStorage/LocalStorage';
```
Comment (fszwedo) [inline Src/Logic/SetProfileInfo/SetProfileInfo.ts:2]: this line seems obsolete atm

**Review by fszwedo — APPROVED** [review]
I have just one suggestion regarding the unused import, apart from that all good

File: Src/Logic/SetProfileInfo/SetProfileInfo.ts (hunk 1 above)
Comment (skorpiom) [reply Src/Logic/SetProfileInfo/SetProfileInfo.ts]: corrected

**Review by robert-czwartosz — APPROVED** [review]

**Comment by robert-czwartosz** [comment]
Looks very good!

**Pull request merged**